client/file_logging.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+
+import king_phisher
+import king_phisher.client.plugins as plugins
+import king_phisher.client.gui_utilities as gui_utilities
+
+# logger name value
+LOGGER_NAME = ''
+
+class Plugin(plugins.ClientPlugin):
+	authors = ['Zach Janice']
+	title = 'Logger'
+	description = """
+	Keep logs of campaign feedback and results. The directory
+	of the logged file(s), the file size (in MB) of the log, and
+	the number of log files kept can be specified. Plugin requires
+	the 'logging' and 'os' packages available for Python.
+	"""
+	req_packages = {}
+	homepage = 'https://github.com/securestate/king-phisher-plugins'
+	options = []
+
+	# this is the primary plugin entry point which is executed when the plugin is enabled
+	def initialize(self):
+		# ensure the directory for the logs exists
+		log_dir = king_phisher.client.application.USER_DATA_PATH
+		if not os.path.exists(log_dir):
+			os.mkdir(log_dir)
+
+		# sanitize the log file size option argument
+		log_file_size = self.config['log_file_size']
+		if log_file_size < 1:
+			unsanitary_file_size_arg = True
+			self.config['log_file_size'] = 1
+			log_file_size = 1
+		else:
+			unsanitary_file_size_arg = False
+
+		# sanitize the log file count option argument
+		log_file_count = self.config['log_file_count']
+		if log_file_count < 1:
+			unsanitary_file_count_arg = True
+			self.config['log_file_count'] = 1
+			log_file_count = 1
+		else:
+			unsanitary_file_count_arg = False
+
+		# convert the specified log file size (MB) to bytes for use by the logger
+		file_size = log_file_size * 1024 * 1024
```

Comment:
`log_file_size` should probably be a constant defined at the top with a comment that it's in MB.